bay_trimesh/anaconda-project.yml

```diff
@@ -0,0 +1,53 @@
+name: bay_trimesh
+description: Visualizing water depth into the Chesapeake and Delaware Bays
+maintainers:
+- philippjfr
+
+channels:
+- pyviz
+
+dependencies: &deps
+- colorcet ==2.0.1
+- datashader ==0.7.0
+- geoviews ==1.6.2
+- holoviews ==1.12.2
+- notebook ==5.7.8
+- pandas ==0.24.2
+
+packages: *deps
+
+platforms:
+- linux-64
+- osx-64
+- win-64
+
+commands:
+  notebook:
+    notebook: bay_trimesh.ipynb
+    env_spec: pyviz-topics-bay-trimesh
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
+  lint:
+    unix: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+variables: {}
+downloads:
+  DATAFILE:
+    url: https://s3.amazonaws.com/datashader-data/Chesapeake_and_Delaware_Bays.zip
+    filename: data
+    unzip: true
+
+env_specs:
+  pyviz-topics-bay-trimesh:
+    channels: []
+    packages: []
+  test:
+    channels:
+    - pyviz
```

Comment:
Similarly, is this declaration needed here given that the following is declared at the top of the file?

```
channels:
- pyviz
```

Reply:
What was the resolution here?

Reply:
I misread. I'll check now.